crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -572,16 +572,22 @@ impl<T: R2D2Connection + 'static> PgIndexerStore<T> {
             .start_timer();
         let mut mutated_objects: Vec<StoredHistoryObject> = vec![];
         let mut object_versions: Vec<StoredObjectVersion> = vec![];
-        let mut deleted_object_ids: Vec<StoredDeletedHistoryObject> = vec![];
+        let mut deleted_objects: Vec<StoredDeletedHistoryObject> = vec![];
+        let mut full_history_objects: Vec<StoredFullHistoryObject> = vec![];
```

Comment:
objects_history is one of the bottlenecks today, instead of adding more workload here, let's start a new parallel task for this here?
https://github.com/MystenLabs/sui/blob/main/crates/sui-indexer/src/handlers/committer.rs#L147

Reply:
Done.